docs/events.md

```diff
@@ -63,6 +64,7 @@ There are several event types that you can record within the customer journey:
 | [`sessionAttributesChanged`](#sessionattributeschanged) | Miscellaneous | Yes           | Records session's attributes changes.
 | [`nothingChanged`](#nothingchanged)                     | Miscellaneous | Yes           | Records a period of inactivity.
 | [`eventOccurred`](#eventOccurred)                       | Miscellaneous | No            | Records a custom event.
+| ['linkOpened'](#linkOpened)                             | Miscellaneous | No            | Records a opened link.
```

Comment:
To keep the code standard, please use `Records a link open.` (similar to the `pageOpened` event).

Reply:
Also, can you please add this one right after the `postViewed` event, please? The last three events are more general events, It's better if we can keep them together.